src/components/data/DataTable.js

```diff
@@ -359,9 +359,9 @@ const DataTable = props => {
                       }, [
                         h(HeaderCell, [
                           !!columnNamespace && span({ style: { fontStyle: 'italic', color: colors.dark(0.75), paddingRight: '0.2rem' } },
-                            columnNamespace)
-                        ]),
-                        [columnName]
+                            columnNamespace),
```

Comment:
I know it's not your change but while touching it, can we enclose `columnNamespace` in an array like we do for other `children` even when it's a single element? FWIW, it took me additional time for my eyes to parse it because of that.

```suggestion
                            [columnNamespace]),
```

Reply:
Changed in https://github.com/DataBiosphere/terra-ui/pull/3059/commits/aa344dafdbcb3834811979146f87dee702f40c30.